Model-Monitoring/Spark/Data_Validation_Spark.ipynb

```diff
@@ -174,7 +174,7 @@
    ],
    "source": [
     "from pyspark import SparkFiles\n",
-    "spark_dataframe = spark.read.format('parquet').load(\"file:///mounts/shared-volume/shared/airbnb_listings.parquet\")"
+    "spark_dataframe = spark.read.format('parquet').load(\"file:///mounts/shared-volume/shared/ezua-tutorials/current-release/Model-Monitoring/Spark/airbnb_listings.parquet\")"
```

Comment:
Should we include "current-release" path section here?

Reply:
Dataset is actually present in that path

Reply:
same here,revert to earlier whylog image